### DISCUSSION TITLE
월드 내에 삽입한 이미지의 해상도가 안드로이드와 IOS에 차이가 있습니다

### DISCUSSION THREAD

**Discussion by IlsYoungOG** [naverz/zepeto-studio-kor#1161] [5. Q&A]
안녕하세요! 
월드 내에 PNG 이미지를 삽입하였습니다. 
PNG 이미지를 일정 간격으로 번갈아 등장하게 하여 전광판을 게시한 것 같은 느낌을 내고자 하였습니다. 

다른 스크립트를 입력하지 않은 이미지 게시물들은 운영 체제에 상관없이 선명한 이미지로 전달 되었으나, 
해당 전광판 이미지만 아이폰과 안드로이드에서 차이를 보였습니다. 

![아이폰](https://user-images.githubusercontent.com/107122738/184095845-bbff6a30-134b-4f00-a68d-eed4682c44c9.jpg)

아이폰에서는 이미지가 깨져 보이고, 
(아이폰 12미니와, 아이폰 X 에서 테스트 하였습니다)

![안드로이드](https://user-images.githubusercontent.com/107122738/184095859-da4e78ad-37cb-41a3-8f76-4a1a9a359977.jpg)

안드로이드(갤럭시S22+)에서는 선명하게 잘 보입니다. 

해당 원인에 대한 이유를 찾고싶습니다. 

감사합니다.

**Comment by IndiaInk10** [comment]
빌드 세팅 > 플레이어 세팅 > 퀄리티 
가보시면 각 OS마다 퀄리티 설정이 다를 것 같습니다!

**Reply by IlsYoungOG** [reply]
OS 마다 동일한 퀄리티인데도 그렇네요 ㅠㅠ 답변주셔서 감사합니다!

**Reply by chlguswls32** [reply]
Texture option 살펴보시면 좋을것같아요. Texture선택후 compression부분이 먼저 확인이 필요해보입니다. Mipmap generate 부분도 확인해보면 좋을것같네요 ㅎㅎ

혹시 월드 llsYoungOG 님께서는 제카엘 님이신가요?

**Reply by IlsYoungOG** [reply]
안녕하세요

> Texture option 살펴보시면 좋을것같아요. Texture선택후 compression부분이 먼저 확인이 필요해보입니다. Mipmap generate 부분도 확인해보면 좋을것같네요 ㅎㅎ
> 
> 혹시 월드 llsYoungOG 님께서는 제카엘 님이신가요?

안녕하세요! 이렇게 알려주셔서서 감사합니다! 한번 시도해보겠습니다 :-)

저희는 제페톤즈(ZEPETONS)라고 합니다!

**Reply by chlguswls32** [reply]
제페톤즈 님이셧군요! 안녕하세요!! ㅎㅎ 
큰 팀이나 회사인가보네요!! ㅎㅎ 월드만들기 열심히 해보려고하는데, 잘 부탁드립니다 :D

**Reply by chlguswls32** [reply]
제페톤즈님 안녕하세요. 혹시 팀이라면 저도 같이 참여하여서 활동할수있는지 문의드립니다.

**Reply by IlsYoungOG** [reply]
> 제페톤즈님 안녕하세요. 혹시 팀이라면 저도 같이 참여하여서 활동할수있는지 문의드립니다.

저희야 너무 좋죠! 
zepetons@naver.com으로 메일 하나 남겨주시면 연락드리겠습니다!

**Reply by chlguswls32** [reply]
메일드렸습니다. 감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 크리에이터님.
텍스쳐의 해상도의 차이가 나는 원인은 다양하지만 확인해볼 수 있는 부분을 안내 드립니다.
댓글에서 언급해주신 것처럼 Quality Setting에 Texture의 해상도 설정이 있는데 
iOS와 안드로이드의 설정이 다르다면 차이가 있을 수 있습니다.
<img width="516" alt="Screen Shot 2022-09-01 at 3 10 20 PM" src="https://user-images.githubusercontent.com/100118650/187845798-47ab3f67-ee4a-489b-b21f-0480ed5b23cb.png">

iOS의 경우에는 이미지의 사이즈가 POT(가로, 세로의 크기가 2의 거듭제곱. 예를 들어 1024, 2048, 4096,...)이 아니면 PVRTC 포맷을 사용하지 않아서 보여지는 퀄리티에 차이가 생길 수도 있습니다.
이미지를 확인했을 때 NPOT (None-POT)라고 뜨면 텍스쳐 압축이 제대로 되지 않습니다. 
![image](https://user-images.githubusercontent.com/100118650/187845924-322897fe-b50b-4c9d-b0c7-0a902b797caa.png)
<img width="437" alt="Screen Shot 2022-09-01 at 3 18 21 PM" src="https://user-images.githubusercontent.com/100118650/187845934-530a82fa-f429-4ecf-b128-63146f201f51.png">

모바일 그래픽스 최적화에 도움이 될만한 유니티 자료 공유 드립니다.
https://blog.unity.com/kr/technology/optimize-your-mobile-game-performance-expert-tips-on-graphics-and-assets

감사합니다.
